mbs_results/growth_rate_output.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import pandas as pd
+
+
+def get_growth_rate_data(filepath):
+    """
+    Filters and pivots wider winsorisation data on period to return growth rate data.
+
+    Parameters
+    ----------
+    filepath : str
+        filepath to the winsorisation output.
+
+    Returns
+    -------
+    pandas.Data.Frame
+        Dataframe containing classification, question number, cell number, and pivoted
+        wider on period with adjusted values.
+    """
+
+    input_data = pd.read_csv(
+        filepath,
+        usecols=[
+            "classification",
+            "question_no",
+            "cell_no_x",
+            "period_x",
+            "adjusted_value",
+        ],
+        dtype={
+            "classification": "Int32",
+            "question_no": "Int8",
+            "cell_no_x": "Int16",
+            "period_x": "Int32",
+            "adjusted_value": "float64",
+        },
+    )
+
+    input_data["period_x"] = (
```

Comment:
There is a helped function in the utils.py module which does the first half of this conversion `convert_column_to_datetime` which would make it easier if we change format in future